modules/packages/Sort.chpl

```diff
@@ -891,6 +896,8 @@ module MergeSort {
 
 pragma "no doc"
 module QuickSort {
+  import Sort.defaultComparator;
+  use super.ShallowCopy;
```

Comment:
I think I'd feel better if it was written `Sort.ShallowCopy`. Does that work?

Reply:
Yep, done now.